### DISCUSSION TITLE
Are there Figma Updates for V9

### DISCUSSION THREAD

**Discussion by fieldju** [microsoft/fluentui#24488] [General] [answered]
I am noticing that the [Figma Web toolkit files](https://developer.microsoft.com/en-us/fluentui#/resources) where last updated in 2020 and are for v7.0.0.

![image](https://user-images.githubusercontent.com/711726/186273992-3575aaa0-e547-4b58-a86e-649e85f0ac00.png)

![image](https://user-images.githubusercontent.com/711726/186273967-1fb261d6-286d-4005-891e-056d17a5df51.png)

![image](https://user-images.githubusercontent.com/711726/186274289-68c5a9bb-2ba7-4805-a604-3983375e5d96.png)

Are there Figma resources available for v9 or a timeline for release?

**Comment by micahgodbolt** [comment, marked as answer]
There is no current plan for a public version of the v9 figma files. Not to say it won't happen at some point, but it's not on our current timeline.
